src/App.jsx

```diff
@@ -14,57 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const clearHandler = () => setSelectedGood('');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const showSelected = () =>
+    selectedGood.endsWith('s')
+      ? `${selectedGood} are selected`
+      : `${selectedGood} is selected`;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {showSelected()}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => clearHandler()}
+          />
+        </h1>
+      ) : (
+        <h1 className="title">No goods selected</h1>
+      )}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={classNames({
+                'has-background-success-light': good === selectedGood,
```

Comment:
Сreate a variable for this clecking and use it everywhere
```suggestion
                'has-background-success-light': isSelected,
```